grails-app/services/org/pih/warehouse/api/MegamenuService.groovy

```diff
@@ -87,6 +87,9 @@ class MegamenuService {
     List buildAndTranslateMenuItems(List menuItems, User user, Location location) {
         def builtMenuItems = []
         menuItems.each {
+            if (it.enabled == false) {
```

Comment:
how is this working? If any menu item is disabled we disable all menu items?

Reply:
@EWaterman only the one that is `enabled: false`. When we return from the `.each` loop, it will continue with the iterations - this `.each` goes through the subsections of a section, hence in this case only the Import Completed Outbound would be hidden

Reply:
yes, exactly like Kacper explained.
This is the same logic we have on nav item sections, we just didn't have this logic for individual items, so this was basically copy and paste

Reply:
hunh interesting. I was assuming the return would break the whole loop and not just be a continue.